lightly_studio/src/lightly_studio/api/routes/api/caption.py

```diff
@@ -4,13 +4,23 @@
 
 from uuid import UUID
 
-from fastapi import APIRouter, Body, Path
+from fastapi import APIRouter, Path
+from fastapi.params import Body
+from pydantic import BaseModel
 from typing_extensions import Annotated
 
 from lightly_studio.db_manager import SessionDep
-from lightly_studio.models.caption import CaptionTable, CaptionView
+from lightly_studio.models.caption import CaptionCreate, CaptionTable, CaptionView
 from lightly_studio.resolvers import caption_resolver
 
+
+class CaptionCreateInput(BaseModel):
```

Comment:
The dataset_id will be removed from CaptionCreate soon (I will start working on it in the afternoon - this includes creating captions in their own dataset).

I would therefore suggest to add a todo to remove this once Caption dataset changes are done.